src/schedlib/policies/stages/build_op.py

```diff
@@ -667,9 +667,15 @@ def get_safe_gaps(block0, block1):
                 t0_parking = t1_parking \
                     = block0.t1 + (block1.t0 - block0.t1) / 2
             else:
-                if get_traj_ok_time(az_parking, az_parking, alt_parking, alt_parking,
-                                    t0_parking) < t1_parking:
-                    raise ValueError("Sun-safe parking spot not found.")
+                safet = get_traj_ok_time(az_parking, az_parking, alt_parking, alt_parking,
+                                         t0_parking)
+                if safet< t1_parking:
+                    alt_parking = self.sun_policy['min_el']
```

Comment:
Elevation of parking spot is changed here.

Reply:
Right, of course!  Sorry I missed that.